src/me/TomTheDeveloper/Kits/CleanerKit.java

```diff
@@ -79,42 +77,45 @@ public void onClean(PlayerInteractEvent event) {
+        if (!(event.getItem().getItemMeta().getDisplayName().contains(ChatManager.colorMessage("Cleaner-Wand-Name"))))
             return;
         if (plugin.getGameAPI().getGameInstanceManager().getGameInstance(event.getPlayer()) == null)
             return;
         if (UserManager.getUser(event.getPlayer().getUniqueId()).isSpectator()) {
-            ChatManager.getSingleMessage("You-Can't-Clean-You're-Spectator", ChatColor.RED + "You can't clean the map now! You are a spectator! You'll respawn at the start of the next wave!");
+            event.getPlayer().sendMessage(ChatManager.colorMessage("You-Can't-Clean-You're-Spectator"));
             return;
         }
         InvasionInstance invasionInstance = (InvasionInstance) plugin.getGameAPI().getGameInstanceManager().getGameInstance(event.getPlayer());
 
         if (UserManager.getUser(event.getPlayer().getUniqueId()).getCooldown("clean") > 0 && !UserManager.getUser(event.getPlayer().getUniqueId()).isSpectator()) {
-            event.getPlayer().sendMessage(ChatManager.getMessage("Ability-Still-On-Cooldown", event.getPlayer(), "clean"));
+        	String msgstring = ChatManager.colorMessage("Ability-Still-On-Cooldown");
+        	msgstring = msgstring.replaceFirst("%COOLDOWN%",Long.toString( UserManager.getUser(event.getPlayer().getUniqueId()).getCooldown("clean")));
+        	event.getPlayer().sendMessage(msgstring);
             return;
         }
         if (invasionInstance.getZombies() != null) {
             for (Zombie zombie : invasionInstance.getZombies()) {
                 if (!plugin.is1_12_R1()) {
                     ParticleEffect.LAVA.display(1, 1, 1, 1, 20, zombie.getLocation(), 100);
-                    System.out.print("ITS SNOT");
                 } else {
                     zombie.getWorld().spawnParticle(Particle.LAVA, zombie.getLocation(), 20, 1, 1, 1);
                 }
                 zombie.remove();
-
             }
             invasionInstance.getZombies().clear();
         } else {
-            event.getPlayer().sendMessage(ChatManager.getSingleMessage("Map-is-already-empty", ChatColor.GREEN + "The map is already empty!"));
+            event.getPlayer().sendMessage(ChatManager.colorMessage("Map-is-already-empty"));
             return;
         }
-        if (plugin.is1_9_R1()) {
+        if (plugin.is1_9_R1() || plugin.is1_12_R1()) {
             event.getPlayer().playSound(event.getPlayer().getLocation(), Sound.ENTITY_ZOMBIE_DEATH, 1, 1);
         } else {
-            // event.getPlayer().playSound(event.getPlayer().getLocation(), Sound.ZOMBIE_DEATH, 1, 1);
+            event.getPlayer().playSound(event.getPlayer().getLocation(), Sound.valueOf("ZOMBIE_DEATH"), 1, 1);
+        }
+        String message = ChatManager.formatMessage(ChatManager.colorMessage("Player-has-cleaned-the-map"), event.getPlayer());
+        for(Player player1 : plugin.getGameAPI().getGameInstanceManager().getGameInstance(event.getPlayer()).getPlayers()) {
```

Comment:
Next time in your for loops. I advice you to use the variable name: "players, player or loopplayer" and not use numbers in it. I type a lot faster if I don't have to type numbers. 
This is not an issue though so don't worry.

Reply:
i just copied and pasted it from ChatManager class so it was the same there :v

Reply:
Oh haha. Yeah it's a preference from me though. And if you copied it from ChatManager than that is my mistake, but for the future on this project, please follow this.
Thanks